cluster/index.js

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2016 Lyteworx LLC.

```

Comment:
Pretty sure you'd need to set the copyright owner to `IBM Corp` not `Lyteworx LLC` if you wanted this to get merged as per the CLA.
Same applies for the headers for all your new files